data/types/activity/physical/physical.go

```diff
@@ -9,11 +9,170 @@ import (
+	ActivityTypeHiking                        = "hiking"
+	ActivityTypeHockey                        = "hockey"
+	ActivityTypeHunting                       = "hunting"
+	ActivityTypeJumpRope                      = "jumpRope"
+	ActivityTypeKickboxing                    = "kickboxing"
+	ActivityTypeLacrosse                      = "lacrosse"
+	ActivityTypeMartialArts                   = "martialArts"
+	ActivityTypeMindAndBody                   = "mindAndBody"
+	ActivityTypeMixedCardio                   = "mixedCardio"
+	ActivityTypeMixedMetabolicCardioTraining  = "mixedMetabolicCardioTraining"
+	ActivityTypeOther                         = "other"
+	ActivityTypeOtherLengthMaximum            = 100
+	ActivityTypePaddleSports                  = "paddleSports"
+	ActivityTypePilates                       = "pilates"
+	ActivityTypePlay                          = "play"
+	ActivityTypePreparationAndRecovery        = "preparationAndRecovery"
+	ActivityTypeRacquetball                   = "racquetball"
+	ActivityTypeRowing                        = "rowing"
+	ActivityTypeRugby                         = "rugby"
+	ActivityTypeRunning                       = "running"
+	ActivityTypeSailing                       = "sailing"
+	ActivityTypeSkatingSports                 = "skatingSports"
+	ActivityTypeSnowboarding                  = "snowboarding"
+	ActivityTypeSnowSports                    = "snowSports"
+	ActivityTypeSoccer                        = "soccer"
+	ActivityTypeSoftball                      = "softball"
+	ActivityTypeSquash                        = "squash"
+	ActivityTypeStairClimbing                 = "stairClimbing"
+	ActivityTypeStairs                        = "stairs"
+	ActivityTypeStepTraining                  = "stepTraining"
+	ActivityTypeSurfingSports                 = "surfingSports"
+	ActivityTypeSwimming                      = "swimming"
+	ActivityTypeTableTennis                   = "tableTennis"
+	ActivityTypeTaiChi                        = "taiChi"
+	ActivityTypeTennis                        = "tennis"
+	ActivityTypeTrackAndField                 = "trackAndField"
+	ActivityTypeTraditionalStrengthTraining   = "traditionalStrengthTraining"
+	ActivityTypeVolleyball                    = "volleyball"
+	ActivityTypeWalking                       = "walking"
+	ActivityTypeWaterFitness                  = "waterFitness"
+	ActivityTypeWaterPolo                     = "waterPolo"
+	ActivityTypeWaterSports                   = "waterSports"
+	ActivityTypeWheelchairRunPace             = "wheelchairRunPace"
+	ActivityTypeWheelchairWalkPace            = "wheelchairWalkPace"
+	ActivityTypeWrestling                     = "wrestling"
+	ActivityTypeYoga                          = "yoga"
+	NameLengthMaximum                         = 100
+	ReportedIntensityHigh                     = "high"
+	ReportedIntensityLow                      = "low"
+	ReportedIntensityMedium                   = "medium"
 )
 
+// Note: ActivityTypes from Apple HealthKit HKWorkoutActivityType
```

Comment:
ahhh ... was having a chuckle to myself thinking "good luck" :)

Reply:
Yep. :)